### ISSUE TITLE
No capacity, balance, or fee details shown in Review channel and Pay now pages

### ISSUE THREAD

**Issue by catch-21** [synonymdev/lsp-ui#87]
The capacity, balance, and fee values are all blank on the Review Channel page.

<img src="https://github.com/user-attachments/assets/ce371229-ee2c-47f3-986d-93013c808a25" width="300">

and the Total amount to pay is blank on the Pay now page. The invoice is still ok though.

<img src="https://github.com/user-attachments/assets/77a8350b-a795-42d1-92c9-976d3bf2a21a" width="300">

This could be due to API change for `/api/channels/<orderId` endpoint

**Comment by dzdidi** [comment]
I see `https://blocktank.synonym.to/api/v1/rate net::ERR_FAILED` in the console and based on these code https://github.com/synonymdev/lsp-ui/blob/master/src/pages/public/confirm/index.tsx#L60-L61 (flag `showFiat`) suggests that exchange rate is to be involved. 
IMO the URL is wrong, do not know where widget takes `https://blocktank.synonym.to/api/v1/rate` from, and can not find it in a code, based on `v1` prefix in the path I would assume it comes from blocktank http client, but the based on env var with which it was build the base should be `https://api1.blocktank.to/api/`, so I do not know. My involvement with widget was quiet minimum until now

**Comment by SeverinAlexB** [comment]
Yes, this is likely related to Rezas old Blocktank server shutdown

**Comment by flaviomoceri** [comment]
@catch-21 please clear caches and try again, we just pushed an update, now I see the values ​​correctly
If it works, please close this error
